src/Illuminate/Foundation/Console/MigrateAutoCommand.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Illuminate\Foundation\Console;
+
+use Doctrine\DBAL\Schema\Comparator;
+use Illuminate\Console\Command;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Filesystem\Filesystem;
+use Illuminate\Support\Facades\Artisan;
+use Illuminate\Support\Facades\Schema;
+
+class MigrateAutoCommand extends Command
+{
+    protected $signature = 'migrate:auto {--fresh} {--seed} {--force}';
+
+    public function handle()
+    {
+        Artisan::call('migrate' . ($this->option('fresh') ? ':fresh' : null) . ($this->option('force') ? ' --force' : null));
+
+        $filesystem = new Filesystem;
+        $dir = base_path('app/Models');
+
+        if ($filesystem->exists($dir)) {
+            foreach ($filesystem->allFiles($dir) as $file) {
+                $class = app('App\\Models\\' . str_replace(['/', '.php'], ['\\', ''], $file->getRelativePathname()));
```

Comment:
What if this is not the namespace

Reply:
See my previous comment. If we added a `model_path` to the `database` config file, we could use that to determine the model path and it's appropriate namespace. Thoughts?